src/universal/styles/ui.js

```diff
@@ -360,7 +355,7 @@ const ui = {
   menuBackgroundColor: '#fff',
   menuBorderColor: appTheme.palette.mid30l,
   menuBorderRadius: borderRadiusSmall,
-  menuBoxShadow: shadow[2],
+  menuBoxShadow: shadow[3],
```

Comment:
Good question! I think that the modal raises the level, but then things would be relative on that new level. Not sure we are going to get perfect, realistic shadow going on, but imagine a modal said “pick a card” (outta 3) and “mark it archived” in a tutorial. The modal is a raised plane, but the cards and their menus would be sitting on top of the level 4 plane, not the base plane. Kinda like my shadow wouldn’t change if I were on top of a mountain or in the valley. It still hits the ground (unless I am on the edge of a cliff). Now if the menu hangs off of the modal, that is weird, but it is equally weird to have a level 5 shadow on a menu over a white modal “why is the menu so high above that modal?” I’d opt for the menu looking right over the modal, not hanging off of the modal.